server/src/routes/api/v1/squidsRouter.js

```diff
@@ -31,11 +32,26 @@ squidsRouter.get(
   })
 );
 
+squidsRouter.get(
+  "/:id",
+  nextWrapper(async (req, res) => {
+    const { id } = req.params;
+    const squid = await Squid.query().findById(id);
```

Comment:
Suggestion: 

await Squid.query().findById(id).throwIfNotFound()

will ensure that you get a non-2xx response if you try to fetch a non-existent squid :) (by throwing an error, which will be caught by nextWrapper)